api/Document.json

```diff
@@ -7267,10 +7273,12 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
```

Comment:
Can you determine the version for this? I've sent https://github.com/foolip/mdn-bcd-collector/pull/590 to create the test for it.